cpp/arcticdb/processing/operation_dispatch_binary.hpp

```diff
@@ -57,7 +57,7 @@ VariantData binary_membership(const ColumnWithStrings& column_with_strings, Valu
             using ColumnTagType = std::decay_t<decltype(column_desc_tag)>;
             using ColumnType =  typename ColumnTagType::raw_type;
 
-            entity::details::visit_type(value_set.base_type().data_type(),[&column_with_strings, &value_set, &func, &output, &column_desc_tag] (auto value_set_desc_tag) {
```

Comment:
I think this was necessary to make another compiler work, but I can't remember which one...

Reply:
Edit: The final solution to please the both compilers was to use catch all by ref